portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase

```diff
@@ -83,4 +83,336 @@ definition {
 		}
 	}
 
+	@description = "LPS-191488. Confirm that it's possible to cancel changes in the Actions tab."
+	@priority = 5
+	test CanCancelEditions {
+		task ("And creates a new item action") {
+			DataSetAdmin.addActions(
+				actionName = "delete",
+				confirmationMessage = "This item was deleted successfully.",
+				icon = "trash",
+				messageType = "Info",
+				type = "Link",
+				url = "http://localhost:8080/o/data-set-manager/");
+		}
+
+		task ("When the user clicks on the new item action 3 dots button") {
+			Click(
+				key_itemName = "delete",
+				locator1 = "AppBuilder#VERTICAL_ELLIPSIS_BUTTON");
+		}
+
+		task ("And clicks on the Edit option") {
+			MenuItem.click(menuItem = "Edit");
+		}
+
+		task ("And changes the Label field content") {
+			Type(
+				key_query = "",
+				locator1 = "ACDefinitions#PRESENT_SEARCH_QUERY_INPUT",
```

Comment:
what has this path "ACDefinitions#PRESENT_SEARCH_QUERY_INPUT" to do with Data Set? I don't see the relation between ACDefinitions and what we are testing here

Reply:
Hmmm in these cases, when I used paths of others components, I didn't know that it was not allowed to do.
Changed!